### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by RobertMrowiec**
# Tasks:
- create integration test for auction-list
- create acceptance test for 

# Testing scenario:
1. run ember test --server
2. choose acceptance login and intergration auction-list test
3. Look how every of 2 (!) tests works :D 

# Preview
image

### CODE REVIEW

File: app/components/auction-list.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ export default Component.extend({
 	infinity: service(),
 	store: service(),
 	
-	async init(){
+	init(){
```
Comment (Rxbsxn) [inline app/components/auction-list.js]: space between method parentheses and opening bracket

File: mirage/factories/auction.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Factory} from 'ember-cli-mirage';
 
 export default Factory.extend({
-  image: 'test',
-  name: 'Sprzedam opla'
+  name: 'Sprzedam złoma',
+  description: 'Panie oddam prawie za darmo bo mnie juz wku... ten samochód',
```
Comment (Rxbsxn) [inline mirage/factories/auction.js]: xd

File: mirage/factories/car.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import { Factory } from 'ember-cli-mirage';
 
 export default Factory.extend({
-	brand: 'Tesla',
-	model: 'S',
+	brand: 'Renault',
+	model: 'Laguna 2',
```
Comment (Rxbsxn) [inline mirage/factories/car.js]: xdd

File: mirage/scenarios/default.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 export default function(server) {
   server.createList('car', 3),
+  // server.createList('auction', 4)
```
Comment (Rxbsxn) [inline mirage/scenarios/default.js]: what about it? :scissors:

File: app/components/login.js [hunk 5]
```diff
@@ -5,13 +5,13 @@ import { inject as service } from '@ember/service'
 export default Component.extend({	
 	auth: service(),
 	store: service(),
+	router: service(),
```
Comment (Rxbsxn) [inline app/components/login.js]: Where do you use the service?

File: app/services/auth.js [hunk 6]
```diff
@@ -9,21 +9,25 @@ export default Service.extend({
 	correctCredentials: null,
 	toastDescription: 'Logged in successfully',
 
-	login(credentials) {
-		return this.get('session').authenticate('authenticator:jwt', credentials)
-		.then(() => this._success())
-		.catch(() => this._fail())
+	async login(credentials) {
+		if ( this.get('session').isAuthenticated ) this._success()
```
Comment (Rxbsxn) [inline app/services/auth.js]: Remove spaces after & before parentheses

File: mirage/factories/auction.js (hunk 2 above)
Comment (RobertMrowiec) [reply mirage/factories/auction.js]: Check this.image, You will know what's going on :D

File: mirage/scenarios/default.js (hunk 4 above)
Comment (RobertMrowiec) [reply mirage/scenarios/default.js]: I'm sure it should go to hairdresser

**Pull request merged**